<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>masterLockerClass</class>
 <widget class="QMainWindow" name="masterLockerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>masterLocker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>620</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #144D46;
background:#08544D;
font-size:16px;
color:#ffffff;
font-family:&quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  border:1px solid #144D46;
  background:#08544D;
  font-family:&quot;微软雅黑&quot;;
  font-size:16px;
  color:#24FFD9;
}
QPushButton:pressed{
margin:1px;
}
</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="qrCodeGenerateWgt" name="qrWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;  全游&lt;/p&gt;&lt;p&gt;营销软件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pcno">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:24px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>大厅01号机</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tip">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:15px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>本机已被锁定，请勿操作！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:60px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>本机已锁</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:26px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;扫码下载幽游一站式&lt;/p&gt;&lt;p&gt;     网吧服务&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hint">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>680</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:24px;
color:#DE0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qrCodeGenerateWgt</class>
   <extends>QWidget</extends>
   <header>qrCodeGenerateWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="masterlocker.qrc"/>
 </resources>
 <connections/>
</ui>
